### DISCUSSION TITLE
Como crio um background com um gradiente e uma cor sólida?

### DISCUSSION THREAD

**Discussion by pxl-developer** [frontendbr/forum#2283] [Perguntas e Respostas] [answered]
Quero criar um background que é composto por um gradiente em cima que vai da esquerda para a direita, e um sólido branco embaixo, pesquisando um pouco encontrei um código que faz isso mas o gradiente fica de cima para baixo, quando mudei da esquerda para a direita o sólido branco também foi para o lado.

Abaixo está a imagem do que tenho em mente.

![iPhone XR, XS Max, 11 – 1](https://user-images.githubusercontent.com/101198158/165544986-05e42ae1-443d-4d07-a5c7-77967ac7f2a6.png)

**Comment by felquis** [comment, marked as answer]
Olá tudo bem? Qual é o seu nome?

O ideal é você enviar o código que você já tem para a gente usa-lo como referência.

### Desafio
> um gradiente em cima que vai da esquerda para a direita, e um sólido branco embaixo

Pode ser que você queria usar múltiplos backgrounds,

O primeiro gradiente é um branco de baixo que começa de cima para baixo com um transparente até 50%
e depois a partir de 50% começa o branco e vai até o final.
```css
  background:
    linear-gradient(to bottom, transparent 50%, white 50%, white 100%);
```

Agora, detalhe para o uso de multiplos backgrounds, que eles são separados por vírgula:
```css
  background:
    linear-gradient(to bottom, transparent 50%, white 50%, white 100%),
```

Ai vem o segundo background, que é o gradiente da parte de cima, que é um gradiente que vai da direita para esquerda começa do branco e terminando no azul.

```css
  background:
    linear-gradient(to bottom, transparent 50%, white 50%, white 100%),
    linear-gradient(to left, white 0%, blue 100%);
```

O resultado que objetivo ficou +- assim:
<img width="453" alt="Captura de Tela 2022-04-27 às 12 57 41" src="https://user-images.githubusercontent.com/736728/165560997-c1d234a3-7dec-466e-942d-22eb8e420a08.png">

codepen https://codepen.io/felquis/pen/wvpVJPP
Guia https://developer.mozilla.org/en-US/docs/web/css/gradient/linear-gradient

**Reply by pxl-developer** [reply]
Funcionou perfeitamente!

Desculpe não colocar um código, acontece que tentei de várias formas e não tinha funcionado então acabei seguindo sem trabalhar no background, mas muito obrigado pela ajuda, realmente me ajudou!
